internal/store/postgres/asset_repository.go

```diff
@@ -431,6 +431,29 @@ func (r *AssetRepository) GetProbesWithFilter(ctx context.Context, flt asset.Pro
 	return results, nil
 }
 
+func (r *AssetRepository) GetAssetDataAttributes(ctx context.Context, urns []string) (map[string]asset.Asset, error) {
+	stmt := sq.Select(`
+			id, urn, name, type, data->'attributes' as data
+	`).From("assets").Where(sq.Eq{"urn": urns})
+
+	query, args, err := stmt.PlaceholderFormat(sq.Dollar).ToSql()
+	if err != nil {
+		return nil, fmt.Errorf("get assets with urns: build query: %w", err)
+	}
+
+	var assets []asset.Asset
```

Comment:
Within the repository layer, we use the `AssetModel` to decode DB representation. Can we do the same here?

Reply:
Will check on this